packages/python27.rb

```diff
@@ -65,5 +65,9 @@ def self.postinstall
     #puts "Upgrading pip...".lightblue
     #system 'pip2 install --upgrade pip'
     #puts
+    if File.exist? "#{CREW_PREFIX}/bin/python3" 
+      FileUtils.rm "#{CREW_PREFIX}/bin/python"
+      FileUtils.ln_s "#{CREW_PREFIX}/bin/python3","#{CREW_PREFIX}/bin/python"
```

Comment:
You could get rid of line 69 with `FileUtils.ln_sf ...`.  You could also make this a one-liner with:
```
FileUtils.ln_sf "#{CREW_PREFIX}/bin/python3", "#{CREW_PREFIX}/bin/python" if File.exist? "#{CREW_PREFIX}/bin/python3"
```
Does it make sense to add this logic in the python3 package also in the hypothetical case where python27 is installed after python3 and `#{CREW_PREFIX}/bin/python` is version 2?

Reply:
I have this as an automatic function in the python3 update I just submitted.

Reply:
How dioes thi look?